pom.xml

```diff
@@ -7,7 +7,10 @@
   <artifactId>livingdocumentation-java</artifactId>
   <version>1-SNAPSHOT</version>
   <packaging>pom</packaging>
-
+  <properties>
+    <sonar.organization>isep-livingdocumentation</sonar.organization>
+    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+  </properties>
```

Comment:
Would it make sense to move these to the YAML file you're adding above, as a `-D` property? They're only really needed in the Sonar build, I think.

Reply:
im not sure since these are the steps SonarCloud provide to add to the pipeline. I will look into it.